src/applications/personalization/dashboard/mocks/appointments/vaos-v2.js

```diff
@@ -1,5 +1,6 @@
-const format = require('date-fns/format');
+// const format = require('date-fns/format');
```

Comment:
probably should just remove this commented out line

Reply:
oops good catch!!